src/Mayaqua/Memory.c

```diff
@@ -3775,6 +3775,9 @@ void Free(void *addr)
 	// Memory release
 	tag->Magic = 0;
 	InternalFree(tag);
+
+	// free() doesn't set the pointer to NULL
+	addr = NULL;

```

Comment:
That's incorrect, as it changes the local variable's value.

I will create a macro in order to set the passed variable's value.